.gitignore

```diff
@@ -14,6 +14,9 @@
 #################################
 /nbproject
 .idea
+/.project
```

Comment:
Please don't app your project/environment specific ignores to cake. These should go to your environment's global gitignore file.